query/src/main/scala/filodb/query/exec/InProcessPlanDispatcher.scala

```diff
@@ -51,6 +53,10 @@ import filodb.query.Query.qLogger
   }
 
   override def isLocalCall: Boolean = true
+
+  override def dispatchStreaming(plan: ExecPlanWithClientParams,
+                                 source: ChunkSource)
+                                (implicit sched: Scheduler): Observable[StreamQueryResponse] = ???
```

Comment:
Would this be involving `` execteStreaming`` on ``ExecPlan``?

Reply:
Correct.